setup/providers/kubernetes-v2/index.md

```diff
@@ -59,19 +59,13 @@ metadata:
  name: spinnaker-role
 rules:
 - apiGroups: [""]
-  resources: ["namespaces", "configmaps", "events", "replicationcontrollers", "serviceaccounts", "pods/logs"]
-  verbs: ["get", "list"]
-- apiGroups: [""]
-  resources: ["pods", "services", "secrets"]
+  resources: ["configmaps", "namespaces", "pods", "secrets", "services"]
```

Comment:
v2 seems to need namespace + delete [here](https://github.com/spinnaker/clouddriver/blob/master/clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/deployer/KubernetesNamespaceHandler.java#L29).